velox/dwio/parquet/reader/ParquetReader.h

```diff
@@ -25,9 +25,6 @@
 
 namespace facebook::velox::parquet {
 
-constexpr uint64_t DIRECTORY_SIZE_GUESS = 1024 * 1024;
```

Comment:
Can you also help replace the values in DWRF reader? 
https://github.com/facebookincubator/velox/blob/8fd31edc70bc6eb465c755e6dfd8b97b16dd98af/velox/dwio/dwrf/reader/ReaderBase.h#L31-L32?